src/resources/js/dependency.js

```diff
@@ -72,6 +75,7 @@
 						}
 					} else {
 						$dependent
+							.hide()
```

Comment:
It does not always work with classes.
We are styling the `.tribe-dependent` to `display: none`; that will not hide an element that is `display` styled with an heavier specificity like:

```html
<div class="wrapper">
    <span class="tribe-dependent some-class">Foo</span>
</div>
```

```css
.wrapper .some-class {
	display: inline-block;
}
```

The element specificity is higher than

```css
.tribe-dependent {
	display: none;
}
```